changelog.txt

```diff
@@ -1,5 +1,18 @@
 # WP Job Manager
 
+## 1.40.0 - 2023-05-11
```

Comment:
Is it this date on purpose because we plan to release this date?

Reply:
That's right :)